server.go

```diff
@@ -172,18 +187,21 @@ Current settings:
 				}
 
 			default:
-				msg = fmt.Sprintf("Sorry, I couldn't process that request: %s", req.Event.Text)
+				subject = fmt.Sprintf("Sorry, I couldn't process that request: %s", req.Event.Text)
 			}
 
-			ts, err := sendMessage("Latest payload stream health report thread", req.Event.Channel, "")
+			ts, err := sendMessage(fmt.Sprintf("Latest payload stream health report thread", lll, lll), req.Event.Channel, thread)
```

Comment:
what are these `lll` variables?

Reply:
oops, pivoted to send subject here with 6474df5 -> f77a5bd